documentation/en/cli-lotus.md

```diff
@@ -0,0 +1,2659 @@
+# lotus
+```
+NAME:
+   lotus - Filecoin decentralized storage network client
+
+USAGE:
+   lotus [global options] command [command options] [arguments...]
+
+VERSION:
+   1.11.0-dev+mainnet
```

Comment:
I'd also drop the `buildType()` from `UserVersion()` here if LOTUS_VERSION_IGNORE_COMMIT is set.